README.md

```diff
@@ -198,6 +189,7 @@ when the service is run.
 | Environment Variable               | Default                   | Description 
 | -----------------------------------|---------------------------|-------------
 | NHAIS_OUTBOUND_SERVER_PORT         | 80                        | The port on which the outbound FHIR REST API will run
+| NHAIS_LOGGING_LEVEL                | DEBUG                     | Application logging level. One of: DEBUG, INFO, WARN, ERROR
```

Comment:
Why DEBUG is default? DEBUG can spit out sensitive data. Let's change that to INFO